airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/source.py

```diff
@@ -76,7 +76,9 @@ class Config:
         default_factory=pendulum.now,
     )
 
-    include_deleted: bool = Field(default=False, description="Include data from deleted campaigns, ads, and adsets.")
+    fetch_thumbnail_images: bool = Field(default=False, description="In each Ad Creative, fetch the thumbnail_url and store the result in thumbnail_data_url")
```

Comment:
Is there any documentation that we can link to from Facebook that demonstrates how this works?